src/main/scala/com/fulcrumgenomics/umi/ExtractUmisFromBam.scala

```diff
@@ -76,7 +76,8 @@ class ExtractUmisFromBam
   @arg(flag = "r", doc = "The read structure, one per read in a template.")      val readStructure: Seq[String],
   @arg(flag = "b", doc = "SAM tags in which to store the molecular barcodes (one-per segment).")
                                                                                  val molecularBarcodeTags: Seq[String] = Seq.empty,
-  @arg(flag = "a", doc = "Annotate the read names with the molecular barcodes. See usage for more details.") val annotateReadNames: Boolean = false
+  @arg(flag = "a", doc = "Annotate the read names with the molecular barcodes. See usage for more details.") val annotateReadNames: Boolean = false,
+  @arg(flag = "x", doc = "The SAM tag with clipping information.") val clippingAttribute: Option[String] = None
```

Comment:
1 & 2 - Done.
3 - I think this tool should be run right after `SamToFastq` and `MarkIlluminaAdapters`, and I don't see any other tags that should be adjusted.  My only thought is that the default should be "XT", but right now I don't know of a way to clear a command line `Option` option.